### ISSUE TITLE
Make the plugin work with C++ files 

### ISSUE THREAD

**Issue by groutoutlook** [AIDK/Highlighter#3]
Such as `.cpp`, `.hpp` extensions, etc...
I dont know why keywords could only be highlighted while editting on `.cs` file though, but if it could also work within the C++ projects, much help for me.

Thanks in advance.

**Comment by sascharo** [comment]
I agree. I thought the extension is broken but I guess it simply doesn't work with `.h` and `.cpp` files.

**Comment by groutoutlook** [comment]
@sascharo  hi, so I just check out your reply to my one year old issue today, I think it's the time to close it since the owner of this repo is inactive.

If you dont mind to build it yourself, I suggest you clone it from my tree. I change virtually nothing but one line of code for the extension to discover C/C++ contents.

[Work like a charm.](https://github.com/groutoutlook/HighlighterVS/blob/eefddfad5c3ce3691105140fcc2d74ccc690c132/Highlighter/Provider.cs#L8)
